docs/Commands.md

```diff
@@ -40,6 +40,11 @@ FT.CREATE idx SCHEMA name TEXT SORTABLE age NUMERIC SORTABLE myTag TAG SORTABLE
 * **NOOFFSETS**: If set, we do not store term offsets for documents (saves memory, does not
   allow exact searches or highlighting). Implies `NOHL`.
 
+* **TEMPORARY**: Create a temporary index that will be expire after a given number of seconds.
```

Comment:
"Create a lightweight temporary index which will expire after the specified period  of inactivity. The internal idle timer is reset whenever the index is searched or added to. Because such indexes are lightweight, you can create thousands of such indexes without negative performance implications"